config.schema.json

```
@@ -21,54 +22,63 @@
               "title": "Id",
               "type": "integer",
               "default": 12345,
+              "description": "ID of service obtained by configuration tool - https://github.com/grzegorz914/homebridge-jablotron-alarm#Identify-Jablotron-services-and-devices.",
```

Just to keep consistency with the release project.
```suggestion
              "description": "ID of service obtained by configuration tool - https://github.com/fdegier/homebridge-jablotron-alarm#Identify-Jablotron-services-and-devices.",
```